models/cutouts.py

```diff
@@ -44,6 +52,12 @@ def format(self, value):
         doc="The source list this cutout is associated with. "
     )
 
+    index_in_sources = sa.Column(
```

Comment:
It occurs to me that there's a possible ambiguity here.  cutouts is associate with a source list, but the source lists on the new, ref, and sub will all have different indexes.  (And, it will often happen that there is no source in the ref's source list at this coordinate.)

I believe what's intended here is that this is the index in the sub's source list, and that that is the same thing that the sources_id points to.  We should make that clear in the documentation here.

(Assuming we're doing to do cutouts from warped ref, `and` that the ref was warped to the new, then the positions in all three images (ref, new, sum) will be the same.)

Reply:
Agreed. I will add some thing in the docstring to make it clear that the sources refer to the subtraction image. 
Maybe it would even be better to call it "detections"?

Reply:
I'm leaving it as `sources` because it is a `SourceList`, and using `detections` would make things much more confusing. I'll just add some emphasis in the docstrings.